umm-spec-lib/src/cmr/umm_spec/validation/spatial.clj

```diff
@@ -32,16 +42,29 @@
     (update-in spatial-extent path (fn [geometry] (map #(assoc % :coord-system coord-sys) geometry)))
     spatial-extent))
 
+(defn- validate-enum
+  "Validate if value exists in enums if value is not nil."
+  [field-path value enums]
+  (when value
+    (when-not (->enum value enums)
+      {field-path
+       [(format "Value (\"%s\") not found in enum (possible values: [%s])"
```

Comment:
Our convention has been to put user input values in brackets in our validation messages.

So `(\"%s\")` should be `[%s]`.

Reply:
done